GPUSimpleVector.h

```diff
@@ -1,5 +1,5 @@
-#ifndef HeterogeneousCore_CUDAUtilities_interface_GPUSimpleVector_h
-#define HeterogeneousCore_CUDAUtilities_interface_GPUSimpleVector_h
+#ifndef GPUSimpleVector_h_
+#define GPUSimpleVector_h_
```

Comment:
Thanks! I only wonder the trailing underscore in the include guards.

Reply:
becaue `input_h` and `output_h` are already used as variable names :-(

Reply:
Ok